### ISSUE TITLE
City and country as a must in affiliations

### ISSUE THREAD

**Issue by marton-balazs-kovacs** [marton-balazs-kovacs/tenzing#75]
For most journals, city and country should be included in the affiliation. Based on the current layout of tenzing it would be quite hard to make this a requirement in the contributors' table. Maybe we can add checks or add tips and tricks info to the app so that users know what to look out for.

**Comment by jcolomb** [comment]
I think it is beyond the scope of this application to tell people how to fill their affiliation information. If we want to go this direction, a website/video instruction might be better than adding checks in the application itself. I am pretty sure what is required may depend on the publisher, too.

**Comment by alexholcombe** [comment]
Yes I think we can close this as it's not clear we should do this. What do you think, @marton-balazs-kovacs ?
